tests/test_create_map.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Test create_map function
+"""
+
+import os
+from rdkit import Chem
+from deepchem import deepchem as dc
+from xymol import XYMOL
+
+
+SMILES = "CCC1(CCC(C)C)C(=O)NC(=O)NC1=O"
+
+metric = dc.metrics.mean_absolute_error
+tasks, datasets, transformers = dc.molnet.load_delaney(featurizer='GraphConv', reload=False)
```

Comment:
Probably we do not have to use deepchem datasets. We can just create a dummy featurizer class/function to take any SMILES and return random vectors, through `featurize` API. Also, the ML model can also be a fake function to mimic real ones, take any vectors and return random prediction, through `predict` API.